IGI.Lab2/src/CarPark.DAL/Models/Accident.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations.Schema;
+using CarPark.DAL.Interfaces;
+
+namespace CarPark.DAL.Models
+{
+    public class Accident : IEntity
+    {
+        public int Id { get; set; }
+
+        [ForeignKey("Contract")]
```

Comment:
I am not sure, but can you try to use `[ForeignKey(nameof(Accident.Contract))]`?

Reply:
Does this make sense?

Reply:
only if you accidentally change the model in the code

Reply:
It's not a big deal.

Reply:
fixed